src/panel/direction/DirectionForm.jsx

```diff
@@ -84,11 +84,12 @@ export default class DirectionForm extends React.Component {
           onChangePoint={(input, point) => this.onChangePoint('destination', input, point)}
           ref={this.destinationRef}
         />
-        <div
-          className="direction-invert-origin-destination icon-reverse"
+        <button
```

Comment:
Great to use a real button, but in this case you need to add a `type="button"` attribute, as the default type of a HTML button is `submit`, which will try to submit the form it belongs too… which will open a new page :D